vita3k/renderer/src/texture_yuv.cpp

```diff
@@ -8,10 +8,35 @@ extern "C" {
 #include <libswscale/swscale.h>
 }
 
+
+
 namespace renderer::texture {
+
+    static SwsContext *renderSwsContext = nullptr;
```

Comment:
```suggestion
    static SwsContext *s_render_sws_context= nullptr;
```
(`s_` only if you keep it `static`)